package.json

```diff
@@ -132,6 +133,7 @@
     "moment-timezone": "^0.5.26",
     "moment-timezone-data-webpack-plugin": "^1.1.0",
     "morgan": "^1.9.1",
+    "object-hash": "^1.3.1",
```

Comment:
does this need to be a dependancy? could it be a devDependancy?

Reply:
Nope, i just chucked it in here for now as I couldn't remember what the situation as Re: https://github.com/bbc/simorgh/issues/2531